### DISCUSSION TITLE
Custom Icon

### DISCUSSION THREAD

**Discussion by cidcore** [ColorlibHQ/AdminLTE#4592] [Q&A]
is there a way to use custom icons not found in any of the libraries?

**Comment by dfsmania** [comment]
Do you mean, to use icons not provided by [font-awesome](https://fontawesome.com/), [Bootstrap Icons](https://icons.getbootstrap.com/), [CoreUI Icons](https://coreui.io/icons/) or any other icon library out there?

If that the case, I suggest you to search for examples on [stack overflow](https://stackoverflow.com/), for example:
- https://stackoverflow.com/questions/38307278/adding-custom-icon-to-html-page

**Comment by cidcore** [comment]
I actually wanted to add our own svg icons that can't be found  in so far any libraries mentioned.

**Reply by dfsmania** [reply]
Well, what prohibits you to inject your SVG icon directly into the html markup? Another alternatives may be:
- https://medium.com/@victoriakronsell/3-different-approaches-to-creating-an-icon-library-using-svgs-1caae8aaee09
- https://stackoverflow.com/questions/11426172/add-custom-icons-to-font-awesome

**Comment by lenamtl** [comment]
Hi,

Another method you can create your own fonticon or sprite using this.
https://fontastic.me/

Fontawesome pro offer a way to add your own icons
https://fontawesome.com/plans
